app/main.py

```diff
@@ -1 +1,18 @@
-# write your code here
+from typing import List, Dict
+
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+def go_to_cafe(friends: List[Dict], cafe: Cafe) -> str:
+    mask_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            return "All friends should be vaccinated"
+        except NotWearingMaskError:
+            mask_to_buy += 1
+    if mask_to_buy > 0:
+        return (f"Friends should buy {mask_to_buy} masks")
```

Comment:
The return statement should not be enclosed in parentheses unless it is intended to return a tuple. You can simply write: `return f"Friends should buy {mask_to_buy} masks"`.